src/Credentials/CredentialProvider.php

```diff
@@ -69,13 +72,18 @@ class CredentialProvider
      */
     public static function defaultProvider(array $config = [])
     {
-        $localCredentialProviders = self::localCredentialProviders();
+        $defaultChain = [
+            self::env(),
+            self::assumeRoleWithWebIdentityCredentialProvider(),
+            self::ini(),
+            self::ini('profile default', self::getHomeDir() . '/.aws/config'),
+        ];
         $remoteCredentialProviders = self::remoteCredentialProviders($config);
```

Comment:
Since the remote and local credential providers are now all interspersed with each other, does this distinction still make sense? Should this just be a single default chain of providers?